forms-html/Shirin-MohammedKhalil/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Shirin-Mohammed</title>
+  </head>
+  <body>
+    <form action="#" method="POST">
+      <!-- Email-->
+      <label for="email">Enter Your E-mail </label><br />
+      <input
+        type="email"
+        id="email"
+        name="email"
+        required
+        placeholder="Shirin-Mohammed@gmail.com"
+      />
+      <br />
+
+      <!-- User Name-->
+      <label for="username"> User Name</label><br />
+      <input
+        type="text"
+        id="username"
+        name="username"
+        placeholder="Shirin - Mohammed"
+      />
+      <br />
+
+      <!-- Password-->
+      <label for="password">Password</label> <br />
+      <input
+        type="password"
+        id="password"
+        name="password"
+        placeholder="***********"
+      />
+      <br />
+
+      <!-- Quantity-->
+      <label for="quantity"> Quantity (between 1 - 5): </label>
+      <input type="number" id="quantity" name="quantity" min="1" max="5" />
+      <br />
+
+      <!-- Volume-->
+      <p>Volume (between 0 and 50):</p>
+      <input type="range" name="volume" min="0" max="50" /> <br />
+
+      <!-- Radio-->
+      <label for="radio"> Radio</label>
+      <br />
+      <input type="radio" id="male" name="gender" value="male" />
+      <label for="male">Male</label><br />
+      <input type="radio" id="female" name="gender" value="female" />
+      <label for="female">Female</label><br />
+      <input type="radio" id="other" name="gender" value="male" />
+      <label for="other">Other</label><br /><br />
```

Comment:
Good job on adding the same name for the radio buttons. This ensures that only one can be selected.